src/frontend/sharedTypes.ts

```diff
@@ -54,3 +54,10 @@ export type CoordinateFormat = 'utm' | 'dd' | 'dms';
 export type DeviceType = 'mobile' | 'desktop';
 
 export type DeviceRole = RoleId;
+
+export const CREATOR_ROLE_ID = 'a12a6702b93bd7ff';
```

Comment:
Maybe worth adding a comment explaining what we're doing here...I don't feel strongly.

Reply:
https://github.com/digidem/CoMapeo-mobile/pull/205/commits/9f2190f2af0529452661b3c5f8fc2548adff86fd